Makefile

```diff
@@ -43,7 +43,8 @@ glideup:
 # the build as a `docker build`.
 build:
 	${DEV_ENV_PREFIX} -e CGO_ENABLED=0 ${DEV_ENV_IMAGE} go build -a -installsuffix cgo -ldflags ${LDFLAGS} -o ${BINARY_DEST_DIR}/boot boot.go || exit 1
-	@$(call check-static-binary,$(BINARY_DEST_DIR)/boot)
+	@$(call check-static-binary,$(BINARY_DEST_DIR)/boot)	
+	${DEV_ENV_PREFIX} ${DEV_ENV_IMAGE} goupx ${BINARY_DEST_DIR}/boot || exit 1
```

Comment:
what's the `|| exit 1` here for? Docker should return non-zero if `goupx` errors inside the container.